restore/action.yml

```
@@ -0,0 +1,26 @@
+name: 'Restore Only Cache'
+description: 'Restore Cache artifacts like dependencies and build outputs to improve workflow execution time'
+author: 'GitHub'
+inputs:
+  path:
+    description: 'The same list of files, directories, and wildcard patterns to restore cache that were used while saving it'
```

Too detailed and may get confusing for an user.

```suggestion
    description: 'A list of files, directories, and wildcard patterns to restore'
```